index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&family=Roboto+Flex:opsz,wght@8..144,500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+    <nav id="toolbar" class="nav-bar">
+        <span id="logo">BBL</span>
```

Comment:
- [ ] Good job using the `<header>` elements for semantics. Consider wrapping your logo in a `<a>` tag that links to the home page as this is best practice and improves user experience. You can check [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more details.

Reply:
I have wrapped my logo with <a> tags